AsyncDisplayKit/ASDisplayNode.h

```diff
@@ -707,6 +711,13 @@ NS_ASSUME_NONNULL_END
 @property (atomic, assign)           BOOL accessibilityViewIsModal;
 @property (atomic, assign)           BOOL shouldGroupAccessibilityChildren;
 
+@property (nonatomic) UIAccessibilityNavigationStyle accessibilityNavigationStyle;
+#if TARGET_OS_TV
+@property(nullable, nonatomic, copy) NSArray *accessibilityHeaderElements;
+#endif
+@property (nonatomic) CGPoint accessibilityActivationPoint;
```

Comment:
Great to have these.  Include , assign for the CGPoint to match CGRect above.  Put the TVOS stuff at the end and group the other ones together.  Let's either list them in the same order Apple does, or group them by variable type.  Feel free to modify the original properties.